tuxemon/monster.py

```diff
@@ -350,11 +350,14 @@ def give_experience(self, amount: int = 1) -> None:
         >>> bulbatux.give_experience(20)
 
         """
```

Comment:
Don't forget to update the docstrings.

The method used to return nothing, but now it returns an int. We need to specify it in the documentation (and explain what's the meaning of the return value).